diagrams/README.md

```diff
@@ -0,0 +1,27 @@
+# Farcaster Node Diagrams
+
+This folder contains a collection of diagrams describing node's components behaviour. Two formats are used: [sequencediagram.org](https://sequencediagram.org/) and [edotor.net](https://edotor.net/).
+
+## [Services' message flow](./sequencediagram.txt)
+
+This diagram presents the message flow between services during a swap execution.
+
+![Services' message flow](./sequencediagram.svg)
+
+## [Reconnect sequence](./reconnect_sequencediagram.txt)
+
+This diagram presents the reconnection mechanism implemented for the peer-to-peer connections.
+
+![Reconnect sequence](./reconnect_sequencediagram.svg)
+
+## [State recovery sequence](./staterecovery_sequencediagram.txt)
+
+This diagram presents the state recovery mechanism. During a swap checkpoints are created, it is possible after a node restart to load the latest registered checkpoint for a given swap and recover its state.
+
+![State recovery sequence](./staterecovery_sequencediagram.svg)
+
+## [State machine](./swap_state_machine.txt)
```

Comment:
Please add a link to edotor.net as done before.

Reply:
There is a link at the top with the sequence diagram editor but I added it in the sentence too in [a218f92](https://github.com/farcaster-project/farcaster-node/pull/703/commits/a218f923d9b45f7ab3cc296078c23f715095df2f)

Reply:
Damn, sorry, don't know how I missed this.